homeassistant/components/climate/radiotherm.py

```diff
@@ -30,6 +31,10 @@
 DEFAULT_AWAY_TEMPERATURE_HEAT = 60
 DEFAULT_AWAY_TEMPERATURE_COOL = 85
 
+OPERATION_LIST = [STATE_AUTO, STATE_COOL, STATE_HEAT, STATE_OFF]
+CT30_FAN_OPERATION_LIST = [STATE_ON, STATE_AUTO]
+CT80_FAN_OPERATION_LIST = [STATE_ON, "Circulate", STATE_AUTO]
```

Comment:
For better consistency, please define your own `STATE_CIRCULATE` constant (in this file) and use it everywhere instead of the string. Reduces the impact of typos and is more in line what all other components do.

Reply:
Done.